rep-2000.rst

```diff
@@ -547,6 +547,126 @@ Build System Support:
+| Poco        |     1.9.2      |     1.9.0     |     1.8.0*     |      1.9.0     |        1.9.4        |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Python      |     3.8.0      |     3.7.3     |     3.8.0      |      3.7.3     |  3.8.2 / 3.7.5****  |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Qt          |     5.12.5     |     5.12.3    |     5.10.0     |      5.11.3    | 5.14.1 / 5.12.5**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+|                              |         **Linux only**                                                |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| PCL         |     1.10.0     |     N/A       |     N/A        |      1.9.1     |       1.10.0        |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+|                                **RMW DDS Milddleware Providers**                                     |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Connext DDS |                              5.3.1***           |               N/A                    |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Cyclone DDS |                                 0.5.1 (Eusebius)                                       |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+|  Fast-RTPS  |                                      1.10.0                                            |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+
+" * " means that this is not the upstream version (available on the official Operating System repositories) but a package distributed by OSRF or the community (package built and distributed on custom repositories).
+
+" ** " Rolling distributions will see multiple version changes of these dependencies during their lifetime.
+The versions shown for OpenEmbedded are those provided by the 3.1 Dunfell release series; the versions provided by the other supported release series are listed here: https://github.com/ros/meta-ros/wiki/Package-Version-Differences .
+Note that the OpenEmbedded releases series for which a ROS distro has support will change during its support time frame, as per the OpenEmbedded support policy shown here: https://github.com/ros/meta-ros/wiki/Policies#openembedded-release-series-support .
+However, it will always be supported by least one stable OpenEmbedded release series.
+
+" \*** " This may be increased to Connext DDS 6.0.0 pending contributions [7]_.
```

Comment:
It doesn't look like this is going to happen, so remove this now.

Reply:
Agreed, removed in 701fbf9